lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -1009,7 +1019,15 @@ class _PointerAdapter extends _BaseAdapter with _WheelEventListenerMixin {
       }
     });
 
-    // Why `domWindow` you ask? See this fiddle: https://jsfiddle.net/ditman/7towxaqp
+    // Move event listeners should be added to `_globalTarget` instead of
+    // `_viewTarget`. This is because `_viewTarget` (the root) captures pointers
+    // by default, meaning a pointer that starts within `_viewTarget` continues
+    // sending move events to its listener even when dragged outside.
```

Comment:
I haven't revisited this yet, BUT, there's a way to have the behavior that we needed _globalTarget for from the _viewTarget!!

See this: https://developer.mozilla.org/en-US/docs/Web/API/Element/setPointerCapture

Reply:
Yeah, that's what I thought as well. Maybe you can change it in a future PR :)

Reply:
I've added a TODO with issue https://github.com/flutter/flutter/issues/157968